subprojects/logging/src/main/java/org/gradle/internal/logging/progress/ProgressLogger.java

```diff
@@ -117,10 +117,13 @@
      */
     void completed();
 
+    void completed(String status);
```

Comment:
Good question. It was only used in one place for the "WAITING" status. But that can also use the new `completed(String, boolean)`. 

I removed the `completed(String)`.